distros/ros2-overlay.nix

```diff
@@ -149,7 +149,7 @@ rosSelf: rosSuper: with rosSelf.lib; {
     postFixup ? "", ...
   }: {
     postFixup = postFixup + ''
-      wrapQtApp "$out/bin/rqt_robot_monitor"
+      wrapQtApp "$out/lib/rqt_robot_monitor/rqt_robot_monitor"
```

Comment:
The same should be done to other `rqt_*` apps. I tested that `ros2 run rqt_graph rqt_graph` doesn't work without similar change.